tests/ZendTest/Session/Service/SessionManagerFactoryTest.php

```diff
@@ -85,6 +88,8 @@ public function testFactoryWillAddValidatorViaConfiguration()
         $this->services->setService('Config', $config);
         $manager = $this->services->get('Zend\Session\ManagerInterface');
 
+        $manager->start();
```

Comment:
Please see the previous discussion: https://github.com/zendframework/zf2/pull/7094#issuecomment-68605430

That is why I said it is a BC break, because the validators are now attached *after* the session has started. This makes sense, because *before* the session has not started there is no session data to compare with and thus the validators always fail.

In my opening post https://github.com/zendframework/zf2/pull/7128#issue-54595275 I explained the tests always passed *before* the security fix and always fail *after* the security fix. The tests added to detect the security fix still pass, which shows this PR does not compromise the security fix in any way.

This test is not related to the security issue, because:
1. If it was here to detect the security issue, it should have failed before the security issue was even addressed. It did not and it always passed the test.
2. The commit which fixed the security issue changed nothing about this test: https://github.com/zendframework/zf2/commit/ddbf43ac3fe28fe98a4104993d0cb4bffb13a026